hgloves/src/components/computer/email/email.js

```diff
@@ -0,0 +1,178 @@
+//Imports
+import React, { Component } from 'react';
+
+import {Editor, EditorState, RichUtils} from 'draft-js';
+// import {stateToHTML} from 'draft-js-export-html';
+
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+
+import './email.css'
+import { Icon } from '@material-ui/core';
+
+import CodeIcon from '@material-ui/icons/Code';
+import FormatItalicIcon from '@material-ui/icons/FormatItalic';
+import StrikethroughSIcon from '@material-ui/icons/StrikethroughS';
+import FormatUnderlinedIcon from '@material-ui/icons/FormatUnderlined';
+import FormatBoldIcon from '@material-ui/icons/FormatBold';
+
+class EmailComponent extends Component {
+
+	state = {
+		editorState: EditorState.createEmpty(),
+		mailObject: '',
+		emailAdress: '',
+		snackbarParam: {
+			open: false,
+			vertical: 'bottom',
+			horizontal: 'center',
+			content: '',
+			error: false,
+		},
+	}
+
+	editorHandle = status => {
+		this.setState({
+			editorState: status,
+		});
+	}
+
+	handleKeyCommand = (command, editorState) => {
+		const newState = RichUtils.handleKeyCommand(editorState, command);
+
+		if (newState)
+			this.editorHandle(newState);
+	}
+
+	sendMail = () => {
+		var formData = {
+			from : this.state.emailAdress,
+			subject: this.state.mailObject,
+			html: this.editor.editor.innerHTML
+
+		};
+		const encodeFormData = (data) => {
+			return Object.keys(data)
+				.map(key => encodeURIComponent(key) + '=' + encodeURIComponent(data[key]))
+				.join('&');
+		};
+		fetch('http://localhost:3300/contact/mail/', {
+			method: 'POST',
+			headers: {
+				'Content-Type': 'application/x-www-form-urlencoded',
+				'Access-Control-Allow-Origin': '*',
+				'Accept': 'application/json',
+				'Access-Control-Allow-Methods': 'GET, POST, PUT, OPTIONS'
+			},
+			body : encodeFormData(formData),
+		})
+	}
+
+	_onBoldClick = () => {
```

Comment:
The '_' at the beginning of the function name indicate that this function will be used only by this component ? It could be nice to make this a norm for all of us ?

Reply:
Maybe, I dont know, we could discuss it in our meeting.